Library/Homebrew/download_strategy.rb

```diff
@@ -652,7 +652,7 @@ def update_submodules
 
   def checkout_submodules(dst)
     escaped_clone_path = cached_location.to_s.gsub(/\//, '\/')
-    sub_cmd = "git checkout-index -a -f --prefix=#{dst}/${toplevel/#{escaped_clone_path}/}/$path/"
+    sub_cmd = "git checkout-index -a -f --prefix=#{dst}/${toplevel/#{escaped_clone_path}/}/\'$path\'/"
```

Comment:
I don't think we want to shell escape this. This string is passed directly to git, and git eventually passes it to the shell where `$path` and `$toplevel` are populated.

I was just pointing out that the `\` is not strictly necessary here.